src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -349,7 +349,7 @@ class ActiveFocusInvestigation extends React.Component<
       {
         Header: STATUS_HEADER,
         accessor: 'status',
-        maxWidth: 60,
+        maxWidth: 100,
```

Comment:
Increase this because the text looked squeezed

Reply:
can we get a screenshot of this for Thai and English?

Reply:
both before and after?

Reply:
after

Reply:
**Thai Before**
![th-before](https://user-images.githubusercontent.com/26719807/106874860-1cb47880-66e7-11eb-8aca-030adb56f426.png)

**English Before**
![en_before](https://user-images.githubusercontent.com/26719807/106874871-2047ff80-66e7-11eb-9906-3598f51909f4.png)

**Thai After**
![th-after](https://user-images.githubusercontent.com/26719807/106875011-42da1880-66e7-11eb-9465-3640421b18f3.png)

**English After**
![en_after](https://user-images.githubusercontent.com/26719807/106875029-466d9f80-66e7-11eb-94d8-596a8953257a.png)